src/dataAPI/ProfileModel.ts

```diff
@@ -151,9 +151,13 @@ export class ProfileModel {
             if (NUMERICS.has(col_type)) {
                 const chartData = await this.executor.getQuantBinnedData(dfName, col_name);
                 const statistics = await this.executor.getQuantMeta(dfName, col_name);
+                const fact = await this.executor.textFormat(dfName, col_name,"N");
+
 
                 cd.summary.statistics = statistics;
                 cd.summary.histogram = chartData;
+                cd.summary.textfact = fact;
+                //console.log(cd.summary.textfact.description)
             } else if (TIMESTAMPS.has(col_type)) {
```

Comment:
Can we add a basic fact for timestamps so that we cover all the main data types?